test/blackbox/common/DDSBlackboxTestsOwnershipQos.cpp

```diff
@@ -2168,106 +2168,51 @@ TEST_P(OwnershipQos, exclusive_kind_keyed_besteffort_disposing_instance)
-    auto middle = std::next(generated_data.begin(), 10);
-    std::list<KeyedHelloWorld> low_strength_data(generated_data.begin(), middle);
-    std::list<KeyedHelloWorld> high_strength_data(middle, generated_data.end());
-    auto expected_data = high_strength_data;
-
-    // Initialize writers.
-    low_strength_writer.ownership_strength(3)
-            .history_kind(eprosima::fastdds::dds::KEEP_ALL_HISTORY_QOS)
-            .reliability(eprosima::fastdds::dds::RELIABLE_RELIABILITY_QOS)
-            .init();
-    ASSERT_TRUE(low_strength_writer.isInitialized());
-
-    // High strength writer will use a custom transport to ensure its data is received after the low strength data.
-    auto test_transport = std::make_shared<test_UDPv4TransportDescriptor>();
-    std::atomic<bool> drop_messages(false);
-    test_transport->messages_filter_ = [&drop_messages](eprosima::fastdds::rtps::CDRMessage_t&)
-            {
-                return drop_messages.load();
-            };
-    high_strength_writer.ownership_strength(4)
-            .history_kind(eprosima::fastdds::dds::KEEP_ALL_HISTORY_QOS)
-            .reliability(eprosima::fastdds::dds::RELIABLE_RELIABILITY_QOS)
-            .disable_builtin_transport()
-            .add_user_transport_to_pparams(test_transport)
-            .init();
-    ASSERT_TRUE(high_strength_writer.isInitialized());
-
-    // Initialize reader.
-    reader.ownership_exclusive()
-            .history_kind(eprosima::fastdds::dds::KEEP_ALL_HISTORY_QOS)
-            .reliability(eprosima::fastdds::dds::RELIABLE_RELIABILITY_QOS)
-            .init();
-    ASSERT_TRUE(reader.isInitialized());
-
-    // Wait for discovery.
-    low_strength_writer.wait_discovery();
-    high_strength_writer.wait_discovery();
-    reader.wait_discovery(std::chrono::seconds::zero(), 2);
-
-    // Send high strength data first, so it has the lowest source timestamps, but drop the messages, so they arrive
-    // later to the reader.
-    drop_messages.store(true);
-    high_strength_writer.send(high_strength_data);
-    EXPECT_TRUE(high_strength_data.empty());
-
-    // Send low strength data, so it has the highest source timestamps.
-    low_strength_writer.send(low_strength_data);
-    EXPECT_TRUE(low_strength_data.empty());
-    // Wait for the reader to receive the data.
-    EXPECT_TRUE(low_strength_writer.waitForAllAcked(std::chrono::seconds(1)));
-
-    // Let high strength writer send the data.
-    drop_messages.store(false);
-
-    // Wait for the reader to receive the high strength data.
-    EXPECT_TRUE(high_strength_writer.waitForAllAcked(std::chrono::seconds(1)));
-
-    // Make the reader process the data, expecting only the high strength data.
-    // The issue was reproduced by the reader complaining about reception of unexpected data.
-    reader.startReception(expected_data);
-    reader.block_for_all();
-}
-
-/*!
- * This is a regression test for redmine issue 20866.
```

Comment:
Would it be useful if we keep the previous redmine issue reference (20866), plus the new reference (21519)?

Reply:
We only reference the ticket describing the problem (bug tracker), not the one for the PR (deliverable)